source/includes/usage-examples/code-snippets/struct-tag.go

```diff
@@ -32,7 +32,7 @@ func main() {
 
 	var uri string
 	if uri = os.Getenv("MONGODB_URI"); uri == "" {
-		log.Fatal("You must set your `MONGODB_URI' environmental variable. See\n\t https://docs.mongodb.com/drivers/go/current/usage-examples/")
+		log.Fatal("You must set your 'MONGODB_URI' environmental variable. See\n\t https://docs.mongodb.com/drivers/go/current/usage-examples/")
```

Comment:
Question:
Should this URL be
https://docs.mongodb.com/drivers/go/current/usage-examples/#environment-variable instead so that the reader doesn't have to scan through the page to find it?

Reply:
Good catch! I agree that the URL should be updated.